### PR TITLE
update proper threshold for memory_hotplug test case on ppc

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID: 1558339
Signed-off-by: mdeng@redhat.com <mdeng@redhat.com>

### CODE REVIEW

**Comment by MiriamDeng** [comment]
As power guest's crashkernel size is much bigger than x86's,which influences and makes some memory_hotplug test cases fail just because of an improper threshold.whatever,it's necessary for us to change a proper one.Thanks.

**Comment by MiriamDeng** [comment]
@yhong@redhat.com,could you give me a hand to review it ?

**Comment by MiriamDeng** [comment]
I've already confirm threshold with ppc developer and he said it's reasonable.Thanks.

**Comment by MiriamDeng** [comment]
Provides some pass test cases logs here.
...
(44/77) smp_4.4096m.repeat1.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.ppc64le.io-github-autotest-qemu.hotplug_memory.after.guest_reboot.hotplug.backend_file.16M_hugepage.policy_interleave.one: PASS (180.96 s)
 (45/77) smp_4.4096m.repeat1.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.ppc64le.io-github-autotest-qemu.hotplug_memory.after.guest_reboot.hotplug.backend_file.16M_hugepage.policy_interleave.two: PASS (173.34 s)
 (46/77) smp_4.4096m.repeat1.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.ppc64le.io-github-autotest-qemu.hotplug_memory.after.guest_reboot.hotplug.backend_file.16M_hugepage.policy_preferred.one: PASS (183.40 s)
 (47/77) smp_4.4096m.repeat1.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.ppc64le.io-github-autotest-qemu.hotplug_memory.after.guest_reboot.hotplug.backend_file.16M_hugepage.policy_preferred.two: PASS (215.30 s)
 (48/77) smp_4.4096m.repeat1.Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.5.ppc64le.io-github-autotest-qemu.hotplug_memory.after.guest_reboot.hotplug.backend_file.16M_hugepage.no_policy.one: PASS (197.88 s)
...

**Comment by YongxueHong** [comment]
Acked-by: Yongxue Hong <yhong@redhat.com>

File: virttest/utils_test/qemu/__init__.py [hunk 1]
```diff
@@ -679,6 +679,8 @@ def check_memory(self, vm=None):
             vm = self.env.get_vm(self.params["main_vm"])
         vm.verify_alive()
         threshold = float(self.params.get("threshold", 0.10))
+        if "ppc" in self.params.get("vm_arch_name"):
+            threshold = float(self.params.get("threshold", 0.15))
```
Comment (luckyh) [inline virttest/utils_test/qemu/__init__.py]: Please set it into the configuration file and leave the comment about why it needs a little bit more on ppc platform.

**Comment by MiriamDeng** [comment]
Thanks and I will have a look.

**Comment by MiriamDeng** [comment]
Per luckyh,close this one and open another,thanks.
https://github.com/autotest/tp-qemu/pull/1269

**Pull request closed without merging**